src/bvh.jl

```diff
@@ -15,6 +41,18 @@ struct BVHNode{T}
     right_child::Union{Nothing, BVHNode}
 end
 
+"""
+    BoundingVolumeHierarchy
+
+Construct the Bounding Volume Hierarchy from a given Vector of
```

Comment:
You're documenting a type, not a function, so you can't say that you "construct the BVH". "A BVH hierarchy"? 

This piece of documentation might be moved to line 63 (`function BoundingVolumeHierarchy(scene::Vector)`). 

(BTW, why are you using an erased type `Vector`? Can't you be more precise, like `Vector{<:Triangle}`?)